src/main/java/de/ait/secondlife/controllers/OfferController.java

```diff
@@ -217,4 +213,118 @@ public ResponseEntity<ResponseMessageDto> recover(
         return ResponseEntity.ok(
                 new ResponseMessageDto(String.format("Offer with id <%s> recovered successful", id)));
     }
+
+    @PatchMapping("/{id}/draft")
+    @Operation(
+            summary = "Change offer status to Draft",
+            description = "Returning an offer to <i>Draft</i> status if the ad does not comply with site policies. Available only for Admin role and for Offers in <i>Verification</i> status."
+    )
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Successful operation",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class))),
+            @ApiResponse(responseCode = "400", description = "Invalid input",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ValidationErrorsDto.class))),
+            @ApiResponse(responseCode = "403", description = "Forbidden",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class))),
+            @ApiResponse(responseCode = "404", description = "Offer not found",
+                    content = @Content(mediaType = "application/json", schema = @Schema(implementation = ResponseMessageDto.class))),
+    })
+    public ResponseEntity<ResponseMessageDto> draft(
```

Comment:
lets name the methods with some action, like "toDraft" or "transferToDraft", or "changeToDraft", or "transitionToDraftStatus"